src/ui/state.h

```diff
@@ -161,7 +161,7 @@ struct State {
     } skeleton_estimation_parameters;
 
 
-
+    void get_window_size(GLFWwindow* handle, int* width, int* height);
```

Comment:
This function doesn't need to be inside `state.h` since it doesn't actually depend on anything in that class. Let's move it somewhere in the utils directory

Reply:
Maybe `uitls.h`?

Reply:
Actually, the function does depend on `state.h` for the `glfw` dependency library, which is only apparently defined in the ui directory, since its first argument needs the dependency in order to be properly defined. When I tried moving this function to the utils directory (i.e. `utils.h`) and tried adding in the `glfw` dependency, I got errors saying it couldn't be found. In this case, I think it may be better to just leave this function in its current implementation.